### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by jorrinpollard**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (C) 2015-2017 MyEtherWallet LLC
+Copyright (C) 2017-2018 MyEllaWallet
```
Comment (ellaismer) [inline LICENSE.md]: MyEtherWallet LLC has the original copyright. So I think we'd need to keep that. And this probably needs to changed to:

```
Original work Copyright (C) 2015-2017 MyEtherWallet LLC
Modified work Copyright (C) 2017-2018 The Ellaism Project
```

File: README.md [hunk 2]
```diff
@@ -1,235 +1,9 @@
...
-- [Reddit: https://www.reddit.com/r/MyEtherWallet/](https://www.reddit.com/r/MyEtherWallet/)
-- [tayvano (founder) reddit: https://www.reddit.com/user/insomniasexx/](https://www.reddit.com/user/insomniasexx/)
-- [kvhnuke (founder) reddit: https://www.reddit.com/user/kvhnuke/](https://www.reddit.com/user/kvhnuke/)
-- [jordan (cmo) reddit: https://www.reddit.com/user/trogdortb001](https://www.reddit.com/user/trogdortb001)
-- [myetherwallet reddit user: https://www.reddit.com/user/myetherwallet](https://www.reddit.com/user/myetherwallet)
--  MEW ETH Donation Address: 0x7cB57B5A97eAbe94205C07890BE4c1aD31E486A8 (mewtopia.eth)
--  MEW BTC Donation Address: 1MEWT2SGbqtz6mPCgFcnea8XmWV5Z4Wc6
-
-
-### `mercury` is the development branch. gh-pages contains only the smaller dist folder only and is served to MyEtherWallet.com
```
Comment (ellaismer) [inline README.md]: Can we keep this line.

File: README.md [hunk 3]
```diff
@@ -1,235 +1,9 @@
...
-- Our infrastructure ("node") is on AWS. [You can also use your own node.](https://myetherwallet.groovehq.com/knowledge_base/topics/how-can-i-connect-to-a-custom-node)
-- We also provide access to Infura.io & Etherscan.io nodes. Use the dropdown in the top-right.
-
-
-### MyEtherWallet
-
-- MyEtherWallet is a free, open-source, client-side tool for easily & securely interacting with the Ethereum network. As one of the leading providers of Ethereum services, MyEtherWallet equips users with an easy-to-understand and accessible suite of tools for their needs.
-- It was created and is maintained by [kvhnuke](https://github.com/kvhnuke) and [tayvano](https://github.com/tayvano).
-
-#### Features
```
Comment (ellaismer) [inline README.md]: I'd suggest to keep this section.

File: README.md [hunk 4]
```diff
@@ -1,235 +1,9 @@
...
-    - sendMode=[ether | token]
-    - tokenSymbol=[ARC | ICN | MKR | ....]
-    - gasLimit=[number] OR gas=[number]
-    - data=[hex data]
-    - Example 1: https://www.myetherwallet.com/?to=0x7cB57B5A97eAbe94205C07890BE4c1aD31E486A8&value=1&tokenSymbol=REP&gaslimit=50000#send-transaction
-    - Example 2: https://www.myetherwallet.com/?to=0x7cB57B5A97eAbe94205C07890BE4c1aD31E486A8&value=1&gaslimit=23000&data=0x5468616e6b20796f752c204d455720322e30#send-transaction
-
-
-
-### Our Philosophy
```
Comment (ellaismer) [inline README.md]: And keep this section.

File: README.md [hunk 5]
```diff
@@ -1,235 +1,9 @@
...
- - **A learning experience, too**: We want to educate about Ethereum, security, privacy, importance of controlling your own keys, how the blockchain works, and how Ethereum and blockchain technologies enable a better world.
- - **If it can be hacked, it will be hacked**: Never save, store, or transmit secret info, like passwords or keys.
- - **Offline / Client-Side**: User should be able to run locally and offline without issue.
- - **Private**: No tracking!!! No emails. No ads. No demographics. We don't even know how many wallets have been generated, let alone who / what / where you are.
- - **Open source & audit-able**
-
-
-
-
-### Users (non-developers)
```
Comment (ellaismer) [inline README.md]: And keep this section.

File: README.md [hunk 6]
```diff
@@ -1,235 +1,9 @@
...
-In case you are not familiar, you need to keep the entire folder in order to run the website, not just index.html. Don't touch or move anything around in the folder. If you are storing a backup of the MyEtherWallet repo for the future, we recommend just storing the ZIP so you can be sure the folder contents stay intact.
-
-As we are constantly updating MyEtherWallet.com, we recommend you periodically update your saved version of the repo.
-
-
-
-
-
-
-### Developers
```
Comment (ellaismer) [inline README.md]: And keep this section.

File: README_MEW.md [hunk 7]
```diff
@@ -0,0 +1,235 @@
+### [https://www.MyEtherWallet.com](https://www.MyEtherWallet.com)
```
Comment (ellaismer) [inline README_MEW.md]: My suggestion is to keep the README file. An additional file might just confuse people on where to find the documentation.

File: app/includes/footer-disclaimer-modal.tpl [hunk 8]
```diff
@@ -15,7 +15,7 @@
 
         <p class="small"><strong> Translations of MyEtherWallet: </strong> The community has done an amazing job translating MyEtherWallet into a variety of languages. However, MyEtherWallet can only verify the validity and accuracy of the information provided in English and, because of this, the English version of our website is the official text. </p>
 
-        <p class="small"><strong>MIT License</strong> Copyright &copy; 2015-2017 MyEtherWallet LLC</p>
+        <p class="small"><strong>MIT License</strong> Copyright &copy; 2017-2018 MyEllaWallet</p>
```
Comment (ellaismer) [inline app/includes/footer-disclaimer-modal.tpl]: So I think we need to credit the original author.

```
Original work Copyright &copy; 2015-2017 MyEtherWallet LLC, Modified work Copyright &copy; 2017-2018 The Ellaism Project
```

**Review by ellaismer — COMMENTED** [review]
We need to keep the original license, and would suggest to give proper credits.

File: LICENSE.md (hunk 1 above)
Comment (jorrinpollard) [reply LICENSE.md]: Ah that was sloppy. Fixing

**Review by ellaismer — APPROVED** [review]

**Pull request merged**